src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,18 +19,18 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
-        countryService.add(usa);
+        CountryDao country = new CountryDaoImpl(sessionFactory);
+        country.add(usa);
 
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
-        actorService.add(vinDiesel);
+        ActorDao actor = new ActorDaoImpl(sessionFactory);
+        actor.add(vinDiesel);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setActors(List.of(vinDiesel));
-        MovieService movieService = null; // TODO: initialize this instance
-        movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
+        MovieDao movie = new MovieDaoImpl(sessionFactory);
+        movie.add(fastAndFurious);
```

Comment:
You have to make dao calls inside Services. In main, You have to call Service's m,ethods